### PR TITLE
Reapply "Detect if running in Flatpak; patch pythonPath and pgadminFi…

### PR DESCRIPTION
**Pull request by razzeee**
…le accordingly."

This reverts commit dfaea944f024ab90139654957ff0ec8f6c17eb8c.

As nobody knows why that has been reverted or cared to document it.

### CODE REVIEW

**Comment by akshay-joshi** [comment]
> …le accordingly."
> 
> This reverts commit [dfaea94](https://github.com/pgadmin-org/pgadmin4/commit/dfaea944f024ab90139654957ff0ec8f6c17eb8c).
> 
> As nobody knows why that has been reverted or cared to document it.

Hi,

This is not the case as Khushboo already mentioned on the old PR that due to a crash we have reverted.

File: runtime/src/js/misc.js [hunk 1]
```diff
@@ -26,6 +26,10 @@ const createDir = (dirName) => {
   }
 };
 
+export const insideFlatpak = () => {
```
Comment (akshay-joshi) [inline runtime/src/js/misc.js:29]: Remove the **export** keyword from here, and add 
`insideFlatpak: insideFlatpak,` into the module.exports = {...} in the same file.

File: runtime/src/js/pgadmin.js [hunk 2]
```diff
@@ -29,6 +29,10 @@ let pythonPath = misc.getPythonPath();
 let pgadminFile = '../web/pgAdmin4.py';
 let configFile = '../web/config.py';
 
+if (insideFlatpak()) {
```
Comment (akshay-joshi) [inline runtime/src/js/pgadmin.js:32]: Change this line to `if (misc.insideFlatpak()) {`

**Review by akshay-joshi — CHANGES_REQUESTED** [review]

**Review by akshay-joshi — CHANGES_REQUESTED** [review]
Commit message should be appropriate.

File: runtime/src/js/misc.js (hunk 1 above)
Comment (razzeee) [reply runtime/src/js/misc.js]: Changed as suggested

**Pull request merged**